### PR TITLE
8.0 website builder translation sig

### PR DESCRIPTION
**Pull request by simongoffin**
opw: 619786

### CODE REVIEW

**Comment by simongoffin** [comment]
Of course it can. Thanks

File: addons/website_event_sale/static/src/js/website.tour.event_sale.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 (function () {
```
Comment (Gorash) [inline addons/website_event_sale/static/src/js/website.tour.event_sale.js]: don't translate tour in 'test' mode

File: addons/website/static/src/js/website.js [hunk 2]
```diff
@@ -304,14 +304,32 @@
...
+                for (var i = 0; i < keys.length; i++){
+                    treat_node(templates[keys[i]]);
                 }
             }).promise();
         }
         return all_ready;
     };
 
+    function treat_node(node){
+        if(node.nodeType==3) {
```
Comment (Gorash) [inline addons/website/static/src/js/website.js]: "==" -> " === "

File: addons/website_sale/static/src/js/website_sale_tour_buy.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 (function () {
     'use strict';
-
```
Comment (Gorash) [inline addons/website_sale/static/src/js/website_sale_tour_buy.js]: don't translate tour in 'test' mode

File: addons/website_sale_options/static/src/js/website_sale.test.js [hunk 4]
```diff
@@ -14,77 +14,77 @@
 
     openerp.Tour.register({
```
Comment (Gorash) [inline addons/website_sale_options/static/src/js/website_sale.test.js]: don't translate tour in 'test' mode

File: addons/website/static/src/js/website.js [hunk 5]
```diff
@@ -304,14 +304,32 @@
                 if ($('html').data('website-id')) {
                     website.id = $('html').data('website-id');
                     website.session = new openerp.Session();
-                    var modules = ['website'];
-                    return openerp._t.database.load_translations(website.session, modules, website.get_context().lang);
+                    return openerp.jsonRpc( '/website/get_website_installed_module', 'call', {}).then(function (modules) {
```
Comment (Gorash) [inline addons/website/static/src/js/website.js]: Create a controllers "/website/translations" who get the installed module and return the same result of "/web/webclient/translations" to have only one call.

**Pull request merged**